app/src/main/java/com/plaid/linksample/network/LinkSampleRetrofit.kt

```diff
@@ -0,0 +1,22 @@
+package com.plaid.linksample.network
+
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.gson.GsonConverterFactory
+
+class LinkSampleRetrofit {
+
+  companion object {
+    // Modify this value to your PC's IP address if not testing on emulator.
```

Comment:
Might want to add that this url is setup for emulators if you are running the sample server locally